backup/v1.2.1/utils/downloader.py

```
@@ -0,0 +1,760 @@
+            'progress_hooks': [self._progress_hook],
+            'quiet': False,
+            'no_warnings': True,
+        }
+        
+        # Add overwrite option if requested
+        if force_overwrite:
+            ydl_opts['force_overwrites'] = True  # Overwrite existing files
+            ydl_opts['nooverwrites'] = False     # Ensure not skipping existing files
+            ydl_opts['overwrites'] = True        # Try alternative option
+            # Add command line options
+            if not 'postprocessor_args' in ydl_opts:
```

Ruff (0.11.9)

Test for membership should be `not in`

Convert to `not in`

(E713)